plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -790,16 +798,16 @@ public File getSystemVMKeyFile() {
         if (keyFile == null || !keyFile.exists()) {
             keyFile = new File("/usr/share/cloudstack-common/scripts/vm/systemvm/id_rsa.cloud");
         }
-        assert (keyFile != null);
```

Comment:
No need for this check anymore?

Reply:
we don't run with assertion so a regular check or none, I'd say.